Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -520,7 +520,7 @@
 
 - type: cargoBounty
   id: BountyLaserGun
-  reward: 28500
+  reward: 5000
```

Comment:
Assuming you're going with the most efficient way (Practice lasers), you need just over 1 crate of steel, and a crate of glass and plastic to make them. This comes to over 5k.

Reply:
> Assuming you're going with the most efficient way (Practice lasers), you need just over 1 crate of steel, and a crate of glass and plastic to make them. This comes to over 5k.

And also you can ask salvagers to give you some steel, yay!

Reply:
while there isnt as much of an issue with this one as there is the others, having such a large materials sink for what is still a net loss isn't good (why would you do this bounty instead of giving the mats to sci or engie).